packages/core/lib/commands/compile.js

```diff
@@ -93,7 +93,9 @@ const command = {
         );
       }
 
-      return await WorkflowCompile.save(config, compilationOutput);
+      const result = await WorkflowCompile.save(config, compilationOutput);
+      await WorkflowCompile.assignNames(config, result);
```

Comment:
This probably unlikely to be an issue, but should we do something to catch errors here? If not, is there a way to add a test?

Reply:
Hm, not sure how to test it... Maybe expand the scenario test? Is that merged?

I think we want errors because the user will have explicitly enabled @truffle/db - if something fails, it seems to make sense to bubble it up to the user.

Reply:
OK updated this to have a real error message and corresponding test